app/assets/javascripts/pretty-text/engines/discourse-markdown-it.js.es6

```diff
@@ -187,6 +188,26 @@ function setupImageDimensions(md) {
   md.renderer.rules.image = renderImage;
 }
 
+function renderAttachment(tokens, idx, options, env, slf) {
+  const linkOpenToken = tokens[idx];
+  const linkTextToken = tokens[idx + 1];
+  const split = linkTextToken.content.split("|");
```

Comment:
Got to be careful on this:

```
"".split("|") // [""]
```

A safer way is:
```
"".split("|").filter(x => x) // []
```

Reply:
Thanks for the tip. :+1: I'm checking for the length of `split` in the following code so I don't need an empty array.